traffic_ops/app/lib/Schema/Result/Deliveryservice.pm

```diff
@@ -278,11 +278,22 @@ __PACKAGE__->table("deliveryservice");
   default_value: false
   is_nullable: 1
 
+=head2 multi_site_origin_algorithm
```

Comment:
Indeed.
For some reason it was not added in the commit creating multi site origin. 
I tried to close this gap in PR 379 (https://github.com/apache/incubator-trafficcontrol/pull/379).

Would you like me to manually remove the changes from the file myself?